haystack/components/embedders/hugging_face_tei_document_embedder.py

```diff
@@ -81,13 +88,23 @@ def __init__(
         """
         huggingface_hub_import.check()
 
+        if not model and not url:
+            warnings.warn(
+                f"Neither `model` nor `url` is provided. The component will use the default model: {DEFAULT_MODEL}. "
+                "This behavior is deprecated and will be removed in Haystack 2.3.0.",
+                DeprecationWarning,
+            )
+            model = DEFAULT_MODEL
+        elif model and url:
+            logger.warning("Both `model` and `url` are provided. The `model` parameter will be ignored. ")
```

Comment:
Describe this in pydocs so users can understand easily what happens when both are provided?

Reply:
good point!